tests/test_sota_checkpoints.py

```diff
@@ -90,7 +90,21 @@ def get_metric_file_name(model_name: str):
     @staticmethod
     def q_dq_config(config):
         nncf_config = NNCFConfig.from_json(config)
-        nncf_config["export_to_onnx_standard_ops"] = True
+        if "compression" in nncf_config:
```

Comment:
So, if there is no compression in config (FP32 model), "export_to_onnx_standard_ops":True will not applied?

Reply:
Correct, and in fact there is no need to do run the corresponding accuracy check since the ONNX would be the same. Same for non-INT8 models